### PR TITLE
Bugfix/v3.17.x.x/ Repair cost and decision

### PR DESCRIPTION
**Pull request by Szczyrk**

### CODE REVIEW

File: Assets/PatchKit Patcher/Scripts/AppUpdater/AppRepairer.cs [hunk 1]
```diff
@@ -162,17 +159,16 @@ int installedVersionId
...
-            return missingFiles.Concat(invalidSizeFiles);
+            return invalidFiles;
         }
 
         private long CalculateRepairCost(AppContentSummary contentSummary, IEnumerable<FileIntegrity> filesToRepair)
         {
             return filesToRepair
                 .Select(f => contentSummary.Files.FirstOrDefault(e => e.Path == f.FileName))
-                .Sum(f => f.Size);
+                .Sum(f => Math.Max(PACK1_BLOCK_SIZE, f.Size));
```
Comment (genail) [inline Assets/PatchKit Patcher/Scripts/AppUpdater/AppRepairer.cs:171]: I've just checked that the chunk size is always defined inside the content or diff summary. You can access it here using `contentSummary.Chunks.Size`. This means that you can remove that constant.

**Review by genail — CHANGES_REQUESTED** [review]

**Review by witcher112 — CHANGES_REQUESTED** [review]
Please check @genail comments

File: Assets/PatchKit Patcher/Scripts/AppUpdater/AppRepairer.cs (hunk 1 above)
Comment (Szczyrk) [reply Assets/PatchKit Patcher/Scripts/AppUpdater/AppRepairer.cs]: Done

File: Assets/PatchKit Patcher/Scripts/AppUpdater/AppRepairer.cs [hunk 2]
```diff
@@ -33,6 +29,7 @@ public class AppRepairer
         private int _lowestVersionWithContentId;
 
         private const double IncreaseRepairCost = 1.5d;
+        private const long PACK1_BLOCK_SIZE = 524288;
```
Comment (witcher112) [inline Assets/PatchKit Patcher/Scripts/AppUpdater/AppRepairer.cs:32]: Unused constant.

**Review by witcher112 — CHANGES_REQUESTED** [review]

File: Assets/PatchKit Patcher/Scripts/AppUpdater/AppRepairer.cs (hunk 2 above)
Comment (Szczyrk) [reply Assets/PatchKit Patcher/Scripts/AppUpdater/AppRepairer.cs]: done

**Review by witcher112 — APPROVED** [review]

**Pull request merged**